src/Pages/SignUp.tsx

```diff
@@ -37,7 +37,7 @@ function SignUpPage() {
     e.preventDefault();
     try {
       googleSignIn();
-      navigate("/profile");
+      navigate("/profile/:id");
```

Comment:
This will not work. If you want to navigate to profilePage it should be navigate(`/profile/${currentUser?.uid}`)